app/common/builders/order_builder.py

```diff
@@ -0,0 +1,49 @@
+from ...repositories.models import Ingredient, Order, Beverage, Size
+from typing import List
+
+class OrderBuilder:
```

Comment:
What's the reason for the Builder Pattern here?

Reply:
A builder pattern will let us have better readability and business code abstraction, as well as give us flexible object creation in case we need it later on. Moreover it helps with consistency when it comes to objects and its states.